### PR TITLE
Use DESTDIR when installing stubby.yml file

### LINKED ISSUE #26: stubby.yml should be installed under $(DESTDIR)
In https://github.com/getdnsapi/stubby/blob/0cbd46ea6d0da727c42b0890d498366cd2b1ec2c/Makefile.am#L18, the config file is installed in ${stubbyconfdir}, but this folder might not exist:
```
test -f /etc/stubby/stubby.yml || \
	/usr/bin/install -c -m 644 ./stubby.yml.example /etc/stubby/stubby.yml
/usr/bin/install: cannot create regular file '/etc/stubby/stubby.yml': No such file or directory
```

That’s because the folder was created under $(DESTDIR), which is not used here.

### PR DESCRIPTION
**Pull request by saradickinson**
This should fix #26. Please test.

### CODE REVIEW

File: Makefile.am [hunk 1]
```diff
@@ -14,5 +14,6 @@ stubby.yml.windows: ${srcdir}/stubby.yml.example
 	awk "{sub(/$$/,\"\r\")}1" $< > $@
 
 install-data-hook: $(CONFIG_FILE)
-	test -f ${stubbyconfdir}/stubby.yml || \
-		${INSTALL_DATA} $(CONFIG_FILE) ${stubbyconfdir}/stubby.yml
+	$(INSTALL) -m 755 -d $(DESTDIR)$(stubbyconfdir)
```
Comment (ArchangeGabriel) [inline Makefile.am]: Not sure this line is necessary since we already have install the example file there normally.

Comment (ArchangeGabriel) [reply Makefile.am]: Excerpt from build log:
```
 /usr/bin/mkdir -p '/build/stubby/pkg/stubby/etc/stubby'
 /usr/bin/install -c -m 644 stubby.yml.example '/build/stubby/pkg/stubby/etc/stubby'
```

Comment (ArchangeGabriel) [reply Makefile.am]: (`DESTDIR=/build/stubby/pkg/stubby` in my case)

**Comment by saradickinson** [comment]
You are right - removed. Fixes the issue otherwise?

**Pull request merged**